wsl.go

```diff
@@ -374,9 +374,26 @@ func (p *Processor) parseBlockStatements(statements []ast.Stmt) {
 
 		switch t := stmt.(type) {
 		case *ast.IfStmt:
+			checkingErrInitializedInline := func() bool {
+				if t.Init != nil {
```

Comment:
NIT: I'm a fan of [happy-path-to-the-left](https://medium.com/@matryer/line-of-sight-in-code-186dd7cdea88) even though this code base is what it is... But it's never too late to improve! 😄 

I also think your naming of variables and comments makes it clear enough to reduce some code. Maybe a re-write to something like this if you think it's ok and still looks clear?

```go
if t.Init == nil {
    return false
}

// Variables were initialized inline in the if statement
// Let's make sure at least one variable is named err just to be safe
return atLeastOneInListsMatch(p.findLHS(t.Init), p.config.ErrorVariableNames)
```

Reply:
That's great!! I generally try to follow that, but haven't heard it called that. Thanks for sharing that article.